.github/workflows/mercury_bot.yml

```diff
@@ -14,16 +14,18 @@ jobs:
         uses: actions/checkout@v3
         
       - name: Set up Python 3.x Part 1
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
-          python-version: "3.9"
+          python-version: "3.10"
 
       - name: Set up Python 3.x Part 2
         run: |
           # set up python
           python3 -m venv ve1
-          cd scripts && ../ve1/bin/pip3 install -r requirements.txt && cd ..
-          cd scripts && ../ve1/bin/python3 setup.py install && cd ..
+          cd scripts
+          ../ve1/bin/pip3 install -r requirements.txt
+          ../ve1/bin/pip3 install .
+          cd ..
```

Comment:
Note that this is actually not needed as the current directory is not persistent across steps.

Reply:
If you mean the `cd` components, that's probably fair. I just kept it consistent with what's in other files. We recently modified the way these ran in a PR, but I don't remember if we split things into multiple lines or put things all in one line. Either way, I picked one and aligned it across all files that did this.

In the future, I think I'd like to replace this with the `make venv.tools` target, but that involves changing a bunch of paths to the invocations. It was more than I wanted to do in this PR.

I can back this change out if it's more comfortable. It doesn't impact the way this PR should work if I back it out.

Reply:
I meant the `cd ../` at the end of the step. It doesn't persist to the next step.

I'm all for consistency across files and steps :+1: Let's leave it like that for now, it also doesn't do any harm.